standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/dbinstall/rules/Postgres.java

```diff
@@ -63,7 +67,15 @@ public String getInitialJdbcUrl() {
 
   @Override
   public boolean isContainerReady(String logOutput) {
-    return logOutput.contains("database system is ready to accept connections");
+    if (logOutput.contains("PostgreSQL init process complete; ready for start up")) {
+      try (Socket socket = new Socket()) {
+        socket.connect(new InetSocketAddress("localhost", 5432), 1000);
```

Comment:
does this mean that host system may not run a local psql server in case it wants to run this test?

Reply:
I see now that this was "like this" before :)